logger_nop.go

```diff
@@ -8,18 +8,18 @@ var NopLogger Logger = &nopLogger{}
 type nopLogger struct {
 }
 
-func (nopLogger) Error(msg string, fields func(LoggerEntry)) {
+func (nopLogger) Error(_ string, _ func(LoggerEntry)) {
```

Comment:
I think when none of the arguments are used by a function you can leave only their types no need for the omitted assign `_`

Reply:
Oh, it's even better! Thanks for suggestion